lib/logzio-nodejs.js

```diff
@@ -128,9 +156,17 @@ LogzioLogger.prototype.log = function(msg) {
 };
 
 LogzioLogger.prototype._popMsgsAndSend = function() {
-    var bulk = this._createBulk(this.messages);
-    this._debug('Sending bulk #' + bulk.id);
-    this._send(bulk);
+    
+    if (this.protocol === 'udp') {
```

Comment:
I thought about that, but then thought maybe some people will still want to send all the udp messages once every 60 seconds, so their network isn't always sending udp.
If they still want it to send each message they can use `bufferSize=1` - I will add this to the documentation.